.env.example

```
@@ -0,0 +1,12 @@
+POSTGRES_DB=auth
+POSTGRES_USER=auth
+POSTGRES_PASSWORD=auth
+POSTGRES_PORT=54322
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The POSTGRES_PORT key should go before the POSTGRES_USER key

(UnorderedKey)